src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1501,12 +1501,30 @@ void PolicyHandler::OnSnapshotCreated(const BinaryMessage& pt_string) {
     return;
   }
 
-  AppIdURL app_url = policy_manager_->GetNextUpdateUrl(urls);
-  while (!IsUrlAppIdValid(app_url.first, urls)) {
-    app_url = policy_manager_->GetNextUpdateUrl(urls);
+  const uint32_t app_id = GetAppIdForSending();
+  const std::string app_policy_id =
+      application_manager_.application(app_id)->policy_app_id();
+
+  EndpointUrls::const_iterator app_item = std::find_if(
+      urls.begin(),
+      urls.end(),
+      [app_policy_id](const EndpointData& ed) {
+
+        return helpers::Compare<std::string, helpers::EQ, helpers::ONE>(
```

Comment:
Here some extra spaces. It would be nice if you'll adjust clang formatter

Reply:
@KVGrygoriev This is what was noticed by EKuliiev, as I said this line was suggested by check_style.sh

Reply:
@ZhdanovP I'm talking about spaces before "return", not about empty line